coldfront/plugins/qumulo/management/commands/add_qumulo_allocation_attribute_type.py

```diff
@@ -126,3 +126,87 @@ def handle(self, *args, **options):
             is_changeable=True,
             is_unique=False,
         )
+
+        # jprew - here is where I'm adding ITDEV-35273-related attribute types
+        # for the allocation collection
+        AllocationAttributeType.objects.get_or_create(
+            attribute_type=AttributeType.objects.get(name="Yes/No"),
+            name="secure",
+            is_required=True,
+            is_private=False,
+            is_changeable=False,
+        )
+
+        # indicates whether the allocation is subject to being audited
+        AllocationAttributeType.objects.get_or_create(
+            attribute_type=AttributeType.objects.get(name="Yes/No"),
+            name="audit",
+            is_required=True,
+            is_private=False,
+            is_changeable=False,
+        )
+
+        AllocationAttributeType.objects.get_or_create(
+            attribute_type=AttributeType.objects.get(name="Date"),
```

Comment:
We should follow up with Kevin/Caroline on this one - there is a `prepaid_billing_date` and a `billing_startdate` and I don't know the difference...

You have a note in `itsm_to_coldfront_map.yaml` about some service provisions/allocations lacking a `billing_startdate`.  I wonder if that might be why (?)

Reply:
NOTE: `prepaid_billing_date` is the date the user selects to start the prepaid period of time; this should only exist for users who are in or will be in prepaid status

`billing_startdate` - this is the date billing begins *period*, whether monthly, prepaid, whatever.

Reply:
`prepaid_billing_date` needs to be pulled out of the ITSM `comment` field.

Reply:
NOTE:  now that I've thought about it, any allocation being brought over will need to have its billing metadata expressly looked at, so I don't think we need to migrate `prepaid_billing_date`, actually.